bullet_train-outgoing_webhooks/app/jobs/webhooks/outgoing/endpoints_deactivation_job.rb

```diff
@@ -0,0 +1,17 @@
+class Webhooks::Outgoing::EndpointsDeactivationJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    return unless automatic_deactivation_endpoint_enabled?
```

Comment:
```suggestion
    return unless automatic_endpoint_deactivation_enabled?
```

Otherwise it reads like there as some kind of an endpoint responsible for automatic deactivation.